index.bs

```diff
@@ -242,7 +288,13 @@ The <dfn method for=FileSystemHandle>queryPermission(|descriptor|)</dfn> method,
 <div algorithm>
 The <dfn method for=FileSystemHandle>requestPermission(|descriptor|)</dfn> method, when invoked, must run these steps:
 
-1. TODO
+1. Let |result| be [=a new promise=].
+1. Run the following steps [=in parallel=]:
+  1. Run <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=query permission steps=] given |descriptor|.
+  1. If that throws an exception, [=reject=] |result| with that exception and abort.
```

Comment:
"that" is potentially confusing. Maybe "If the previous step throws an exception"

But see below...